src/Symfony/Component/Yaml/Parser.php

```diff
@@ -336,7 +336,8 @@ private function getNextEmbedBlock($indentation = null, $inSequence = false)
                 $removeComments = !preg_match($removeCommentsPattern, $this->currentLine);
             }
 
-            if ($isItUnindentedCollection && !$this->isStringUnIndentedCollectionItem($this->currentLine)) {
+            $isStringUnIndentedShortcutKey = (2 === ($indent - $newIndent));
```

Comment:
@xabbuh the value is hard-coded here https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Yaml/Parser.php#L111 can you see now is it ok ?